components/nfs/component.go

```diff
@@ -181,6 +181,15 @@ func (c *component) Check() components.CheckResult {
 			return cr
 		}
 
+		// safe to clean/delete old files
+		if err := checker.Clean(); err != nil {
```

Comment:
the clean function delete all files, how do you guarantee if the deletion operation does not affect other gpud health check?

Reply:
Discussed offline, will make this much simpler.